--- /usr/share/redmine/app/views/issues/show.html.erb	2016-06-05 10:28:58.000000000 +0300
+++ app/views/issues/show.html.erb	2016-10-29 22:56:10.248756460 +0300
@@ -77,7 +77,7 @@
 <% if @issue.description? %>
 <div class="description">
   <div class="contextual">
-  <%= link_to l(:button_quote), quoted_issue_path(@issue), :remote => true, :method => 'post', :class => 'icon icon-comment' if authorize_for('issues', 'edit') %>
+  <%= link_to l(:button_quote), issue_id_quoted_issue_path(@issue), :remote => true, :method => 'post', :class => 'icon icon-comment' if authorize_for('issues', 'edit') %>
   </div>
 
   <p><strong><%=l(:field_description)%></strong></p>
