--- /Users/colcek/Projects/Ekodenge/workspace/internal/service-redmine/plugins/issue_id/show.html.erb	2020-07-06 13:28:39.000000000 +0300
+++ /Users/colcek/Projects/Ekodenge/workspace/internal/service-redmine/plugins/issue_id/app/views/issues/show.html.erb*	2020-07-06 13:54:27.000000000 +0300
@@ -79,7 +79,7 @@
 <hr />
 <div class="description">
   <div class="contextual">
-  <%= link_to l(:button_quote), quoted_issue_path(@issue), :remote => true, :method => 'post', :class => 'icon icon-comment' if @issue.notes_addable? %>
+  <%= link_to l(:button_quote), issue_id_quoted_issue_path(@issue), :remote => true, :method => 'post', :class => 'icon icon-comment' if @issue.notes_addable? %>
   </div>
 
   <p><strong><%=l(:field_description)%></strong></p>
