--- /usr/share/redmine/app/views/mailer/issue_edit.text.erb	2016-10-10 10:48:13.000000000 +0300
+++ app/views/mailer/issue_edit.text.erb	2017-12-05 18:46:12.385703127 +0200
@@ -1,4 +1,4 @@
-<%= "(#{l(:field_private_notes)}) " if @journal.private_notes? -%><%= l(:text_issue_updated, :id => "##{@issue.id}", :author => @journal.user) %>
+<%= "(#{l(:field_private_notes)}) " if @journal.private_notes? -%><%= l(:text_issue_updated, :id => "##{@issue.to_param}", :author => @journal.user) %>
 
 <% details_to_strings(@journal_details, true).each do |string| -%>
 <%= string %>
